src/test/java/spoon/test/api/FileSystemFolderTest.java

```diff
@@ -19,4 +25,17 @@ public void jarFileIsNotSubfolder() {
 		List<SpoonFolder> subFolders = folder.getSubFolders();
 		assertTrue(subFolders.isEmpty());
 	}
+
+	@Rule
+	public ExpectedException expectedEx = ExpectedException.none();
+
+	@Test
+	public void testLauncherWithWrongPathAsInput() {
+		expectedEx.expect(SpoonException.class);
+		expectedEx.expectMessage("java.io.FileNotFoundException: The following file does not exist: /Users/urli/Github/spoon/src/wrong/direction/File.java");
```

Comment:
Oh. Indeed I was a bit fast in my cut & paste. The intent was only to paste the FileNotFound. I fix that.